base/src/main/java/com/smartdevicelink/proxy/rpc/SubscribeVehicleData.java

```diff
@@ -116,10 +116,13 @@
  * 		<tr>
  * 			<td>externalTemperature</td>
  * 			<td>Boolean</td>
- * 			<td>The external temperature in degrees celsius</td>
+ * 			<td>The external temperature in degrees celsius. This parameter is deprecated starting RPCSpec 7.1, please see climateData.</td>

```

Comment:
@kostyaBoss On line 119, I got 7.1.0 instead of 7.1 using the RPC generator tool.  I would replace the 7.1 with 7.1.0.